discourse_rock/patches/discourse-charm.patch

```diff
@@ -0,0 +1,15 @@
+--- a/lib/tasks/discourse-charm.rake
++++ b/lib/tasks/discourse-charm.rake
+@@ -0,0 +1,12 @@
++# frozen_string_literal: true
++
++desc "Check if a user exists for given email address"
```

Comment:
Can you add a comment here to the upstream PR so that we can keep track of it?

Reply:
https://github.com/discourse/discourse/pull/27163